docs/data-list.md

```diff
@@ -715,13 +810,54 @@ Data abbreviated for brevity.
 
 Data aggregated from the `mostRecent` record for each Wikibase.
 
-All data is paginated as above; the Page Number and Page Size arguments are the same, and the queries return Meta and Data as above.
+### Aggregated Created
+
+First log date of each wikibase, aggregated by year.
+
+#### Example:
+
+Query:
+
+```
+query MyQuery {
+  aggregateCreated {
+    wikibaseCount
+    year
+  }
+}
+```
+
+Result:
+
+```
+{
+  "data": {
+    "aggregateCreated": [
+      {
+        "wikibaseCount": 1,
+        "year": 2005
+      },
+      {
+        "wikibaseCount": 1,
+        "year": 2006
+      },
+      {
+        "wikibaseCount": 1,
+        "year": 2011
+      },
+      ...
+    ]
+  }
+}
+```
+
+Data abbreviated for brevity.
```

Comment:
Sorry, I do not understand what that means. Can you rephrase or put some details? So it is about aggregation of certain values over time? What is the `wikibaseCount`?

Reply:
Ah, now I got it. So it aggregates the creation dates. So the output above shows how many wikibase instances have been created in each year. Is that correct?

Reply:
Exactly